### PR TITLE
[add-values-manifest] add manifests

### PR DESCRIPTION
**Pull request by aliartiza75**

### CODE REVIEW

**Comment by stakater-user** [comment]
@aliartiza75 Yikes! You better fix it before anyone else finds out! [Build 1](https://jenkins.release.stakater.com/job/nordmart-apps/job/stakater-nordmart-catalog/job/PR-30/1/) has Failed!

**Comment by stakater-user** [comment]
@aliartiza75 Yikes! You better fix it before anyone else finds out! [Build 2](https://jenkins.release.stakater.com/job/nordmart-apps/job/stakater-nordmart-catalog/job/PR-30/2/) has Failed!

File: deployment/manifests/mysql-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: nordmart-dev-apps
```
Comment (ahmedwaleedmalik) [inline deployment/manifests/mysql-secret.yaml]: Defaut namespace

**Comment by stakater-user** [comment]
@aliartiza75 Yikes! You better fix it before anyone else finds out! [Build 3](https://jenkins.release.stakater.com/job/nordmart-apps/job/stakater-nordmart-catalog/job/PR-30/3/) has Failed!

**Review by ahmedwaleedmalik — APPROVED** [review]

**Comment by stakater-user** [comment]
@aliartiza75 Yikes! You better fix it before anyone else finds out! [Build 4](https://jenkins.release.stakater.com/job/nordmart-apps/job/stakater-nordmart-catalog/job/PR-30/4/) has Failed!

**Pull request merged**